README.md

```diff
@@ -7,6 +7,25 @@ version for the lifetime of the product.
 
 This repository is solely maintained by Docker, Inc.
 
+:warning: **This repository is now deprecated and will be archived** :warning:
```

Comment:
Maybe put it further up so it's the first thing we see?

Reply:
Done!